alf/examples/hypernet_mnist.gin

```diff
@@ -2,9 +2,8 @@ import alf.algorithms.hypernetwork_algorithm
 import alf.trainers.policy_trainer
 
 # dataset config
-create_dataset.dataset_name='mnist'
-create_dataset.train_batch_size=50
-create_dataset.test_batch_size=100
+datagen.load_mnist.train_bs=50
```

Comment:
Convert this file to _conf.py?

Reply:
I'll leave this for Jerry.

Reply:
Will do this after Neale's PR is merged.